tchannel/tornado/connection.py

```diff
@@ -49,6 +51,13 @@
 log = logging.getLogger('tchannel')
 
 
+#: Sentinel object representing that the connection is outgoing.
+OUTGOING = object()
```

Comment:
A bool is confusing and error-prone... is "True" incoming or outgoing? You'll ask yourself that every time you use it.